src/ch9329.cpp

```diff
@@ -12,14 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
 #include <array>
 #include <cstdint>
+
 #include <libhal-input/ch9329.hpp>
 #include <libhal-util/bit.hpp>
+#include <libhal-util/enum.hpp>
 #include <libhal-util/serial.hpp>
 #include <libhal/serial.hpp>
 #include <libhal/timeout.hpp>
 #include <libhal/units.hpp>
+#include <span>
 
```

Comment:
```suggestion
#include <cstdint>

#include <algorithm>
#include <array>
#include <span>

#include <libhal-input/ch9329.hpp>
#include <libhal-util/bit.hpp>
#include <libhal-util/enum.hpp>
#include <libhal-util/serial.hpp>
#include <libhal/serial.hpp>
#include <libhal/timeout.hpp>
#include <libhal/units.hpp>

```

`#include <cstdint>` is a C header so that goes on top.